spec/components/collections/show/details_header_component_spec.rb

```diff
@@ -8,19 +8,24 @@
 
   context 'with a new, first draft collection' do
     let(:collection_version) { build_stubbed(:collection_version, :first_draft) }
+    let(:collection_id) { collection_version.collection.id }
 
     it 'does not render the spinner' do
       expect(rendered.to_html).not_to include 'fa-spinner'
     end
+
+    it 'renders turbo frames' do
+      expect(rendered.css('turbo-frame')[0]['src']).to eq "/collections/#{collection_id}/deposit_button"
+      expect(rendered.css('turbo-frame')[1]['src']).to eq "/collection_versions/#{collection_version.id}/edit_link"
+    end
   end
 
   context 'with a depositing collection' do
     let(:collection_version) { build_stubbed(:collection_version, :depositing) }
 
-    it 'does not render the spinner' do
+    it 'does renders the spinner' do
```

Comment:
"does renders" is awkward. How about just "renders"?

Reply:
`it 'spinners' do`